english/marshmallow.md

```diff
@@ -22,7 +22,7 @@ vacant eyed young man lounging on a nearby sofa.
 
 [Interrogate the man for clues](./movie-ripoffs/zoolander/zoolander.md)  
 
-These marshmallows are numerous and they look so delicious.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.